packages/react-components/src/atoms/Card.tsx

```diff
@@ -9,7 +9,7 @@ import { paddingStyles } from '../card';
 export type AccentVariant = 'default' | 'red' | 'green';
 
 export const accents: Record<AccentVariant, string> = {
-  default: `border-color: ${colors.silver.rgb};`,
+  default: `background-color: ${colors.paper.rgb}; border-color: ${colors.silver.rgb};`,
```

Comment:
Slightly unfortunate that `Card` loses the ability to be used in a `grey` or `dark` context. Maybe I'll come up with a solution at some point.
Either way:
```js
import {light} from '../theme'
default: `${light} border-color: ...`
```
so that both background-color and color from the light theme are applied, otherwise things get ugly in a dark context (paper on paper)